libtonode-tests/tests/concrete.rs

```diff
@@ -1409,18 +1409,12 @@ mod slow {
             - first_send_to_transparent
             - (3 * u64::from(MARGINAL_FEE));
         assert_eq!(
-            recipient
-                .wallet
-                .shielded_balance::<OrchardDomain>(None, &[])
-                .await,
+            recipient.wallet.pending_balance::<OrchardDomain>().await,
```

Comment:
I believe that this is properly pending balance because the value in question has been calculated without adding a confirming block.

Reply:
yeah. did this test pass before??